config/manager/manager.yaml

```diff
@@ -34,7 +34,7 @@ spec:
               fieldPath: metadata.namespace          
         - name: DEX_OPERATOR_IMAGE
           value: "quay.io/identitatem/dex-operator:latest"
-        image: quay.io/identitatem/idp-mgmt-operator@sha256:f13d0e986746488a5033e16a4dc0e5a0be8e2f3024f72b680cf8709308fd6fd9
+        image: quay.io/identitatem/idp-mgmt-operator@sha256:a6ab15bc688acb1894d965f6b3f5ca68d944a77d40958a1e64a771fd784f1177
```

Comment:
Should this be `latest`?  Should we add a .gitignore rule for this file?

Reply:
yes it should, I will change it